pkg/pxc/app/statefulset/node.go

```diff
@@ -279,7 +279,7 @@ func (c *Node) LogCollectorContainer(spec *api.LogCollectorSpec, logPsecrets str
 			Value: "/var/lib/mysql",
 		},
 		{
-			Name: "POD_NAMESPASE",
+			Name: "POD_NAMESPACE",
```

Comment:
@franklouwers thank you for PR! but we can remove these var only for crVersion >= 1.20.0 like https://github.com/percona/percona-xtradb-cluster-operator/pull/1949/files#diff-4c64a958f53a8f3616564c42beb08dfeeaee83a9ae0a13dab3a41cb375998748L407
We need to have it to avoid cluster restart as soon as you update operator image.

Reply:
@hors I am sorry, I have no idea how to do that. I don't know (nor really want to know) how the operator works. I've opened an issue for an embarrassing typo that got in, about 3 months ago. Last week you asked me to prep a PR. I just did. 

As the typo is in an envvar, which will force a restart, then yes, I assume it will restart. If there's a mechanism to prevent that, I don't know how to do that. It doesn't seem to be in the public documentation. 

You mention crVersion: I see it's currently 1.17.0. Should I just change that to 1.20.0 in all places? Why 1.20 btw and not 1.18?

Reply:
@franklouwers any updates from your side for these comments?

Reply:
@gkech was hoping to quickly get an embarrassing typo out of the way. Didn't know I had to guess version numbers and do strange dances. Closing this PR, as I have no time for this, just tried to get a quick win in terms of usability.